### ISSUE TITLE
Reenable signature workflow extended test

### ISSUE THREAD

**Issue by miminar** [openshift/origin#16344]
The extended signature workflow test, disabled by #15807, used to fail with:

```
/tmp/openshift/build-rpm-release/tito/rpmbuild-originV11XYu/BUILD/origin-3.7.0/_output/local/go/src/github.com/openshift/origin/test/extended/registry/signature.go:112
Expected
    <string>: error verifying signature sha256:248398a9acf9b2f80aabebee81ae88d70768eadebccecd786a0e6e1d2756e4cf@67655a940b7d40e9f4d3067765b613e7 for image sha256:248398a9acf9b2f80aabebee81ae88d70768eadebccecd786a0e6e1d2756e4cf (verification status will be removed): failed to get image "sha256:248398a9acf9b2f80aabebee81ae88d70768eadebccecd786a0e6e1d2756e4cf" manifest: Get http://docker-registry.default.svc:5000/v2/: malformed HTTP response "\x15\x03\x01\x00\x02\x02"
to contain substring
    <string>: identity is now confirmed
/tmp/openshift/build-rpm-release/tito/rpmbuild-originV11XYu/BUILD/origin-3.7.0/_output/local/go/src/github.com/openshift/origin/test/extended/registry/signature.go:106
```

The registry is secured during the extended test run. It looks like the `oadm verify-image-signature` cannot talk to secured registry. This needs to be fixed before re-enabling this test.

**Comment by miminar** [comment]
@mfojtik FYI. Feel free to take this over when you enhance the `verify-image-signature` command with `--insecure` flag.

**Comment by miminar** [comment]
This has been resolved in #17202.

The only problem is that it was removed one week later with a rebase to 1.8.1 in #17115. So the test doesn't exist anymore.

The issue shall be moved to image-registry repository.

**Comment by miminar** [comment]
Re-opened in the openshift/image-registry#74.
